betty/cache/_base.py

```diff
@@ -47,14 +46,7 @@ def __init__(
     ):
         self._scopes = scopes or ()
         self._scoped_caches: MutableMapping[str, Self] = {}
-        self._locks: MutableMapping[str, _Lock] = defaultdict(
-            AsynchronizedLock.threading
-        )
-        self._locks_lock = AsynchronizedLock.threading()
-
-    async def _lock(self, cache_item_id: str) -> _Lock:
-        async with self._locks_lock:
-            return self._locks[cache_item_id]
+        self._lock = AsynchronizedLock.multiprocessing()
```

Comment:
Separate this out into a separate PR

Reply:
Also, benchmark what's better: a single lock, or lazily loading and deleting locks per cache item ID